### ISSUE TITLE
Chaincode Modification

### ISSUE THREAD

**Issue by Aura-Buddy** [vanitas92/fabric-external-chaincodes#15]
Howdy, 

I am wondering how one can modify the chaincode. I was working with fabcar  and modified the fabcar.go file and remade the chaincode image to contain the changes. However, upon interacting with the chaincode from the clients, the changes did not take effect. Is the chaincode defined somewhere else?

**Comment by vanitas92** [comment]
Hey @Aura-Buddy ,

I have also tested the same scenario as you are describing and it DID in fact changed the chaincode behavior when I deployed the new image of the chaincode and no new version increase or update has happened from Hyperledger Fabric. Although I find it a bit dangerous this situation when there are multiple parties, I would strongly recommend to always upgrade the chaincode definition from Fabric and then upgrade the images with the new code.

Hope that this answers your doubt!
